### ISSUE TITLE
bluetooth/bluetooth_obex_send stuck forever

### ISSUE THREAD

**Issue by xanthein** [canonical/checkbox#1289]
### Bug Description

Running bluetooth/bluetooth_obex_send twice make it stuck forever on Realtek 8852BE

```
18:08:28 ---------------------------[ Running job 221 / 293 ]----------------------------
18:08:28 ----------------------------[ Bluetooth OBEX send ]-----------------------------
18:08:28 ID: com.canonical.certification::bluetooth/bluetooth_obex_send
18:08:28 Category: Bluetooth tests
18:08:28 --------------------------------------------------------------------------------
18:08:28 INFO:controller:controller: Waiting for job to finish.
18:08:28 Host:[7C:B2:7D:4B:14:95]
18:08:40 Get available BTDEVADDR:[7C:B2:7D:4B:14:95]
18:08:51 INFO: =================================[ Send test ]==================================
18:08:51 INFO: Using JPEG_Color_Image_Ubuntu.jpg as a test file
18:08:51 INFO: Sending /usr/share/checkbox-provider-base/data/images/JPEG_Color_Image_Ubuntu.jpg to 7C:B2:7D:4B:14:95
18:08:51 INFO: PASS
18:08:51 INFO:controller:controller: Finishing job with a result: None
18:08:51 --------------------------------------------------------------------------------
18:08:51 Outcome: job passed
18:08:51 ---------------------------[ Running job 222 / 293 ]----------------------------
18:08:51 -------------------[ Bluetooth OBEX send after suspend (S3) ]-------------------
18:08:51 ID: com.canonical.certification::after-suspend-bluetooth/bluetooth_obex_send
18:08:51 Category: Bluetooth tests
18:08:51 --------------------------------------------------------------------------------
18:08:51 INFO:controller:controller: Waiting for job to finish.
18:08:51 Host:[7C:B2:7D:4B:14:95]
18:09:03 Host:[28:3A:4D:46:79:C0]
18:09:15 Host:[34:6F:24:A8:93:EE]
18:09:27 Host:[80:32:53:D8:0D:1E]
20:09:36 
20:09:36 ERROR: Output timeout reached! (7200s)
```

### To Reproduce

1. checkbox-cli control <IP> launcher

launcher
```
#!/usr/bin/env checkbox-cli
[launcher]
launcher_version = 1

[test plan]
unit = com.canonical.certification::after-suspend-bluetooth-cert-automated
forced = yes

[test selection]
forced = yes
exclude =  .*cpu/clocktest

[ui]
type = silent

[daemon]
normal_user = ubuntu

[environment]
ROUTERS = multiple
WPA_BG_SSID = cert-bg-wpa-tel-l3-01
WPA_BG_PSK = insecure
WPA_N_SSID = cert-n-wpa-tel-l3-01
WPA_N_PSK = insecure
WPA_AC_SSID = cert-ac-wpa-tel-l3-01
WPA_AC_PSK = insecure
WPA_AX_SSID = cert-ax-wpa-tel-l3-01
WPA_AX_PSK = insecure
WPA3_AX_SSID = cert-ax-wpa3-tel-l3-01
WPA3_AX_PSK = insecure
OPEN_BG_SSID = cert-bg-open-tel-l3-01
OPEN_N_SSID = cert-n-open-tel-l3-01
OPEN_AC_SSID = cert-ac-open-tel-l3-01
OPEN_AX_SSID = cert-ax-open-tel-l3-01
BTDEVADDR = 7C:B2:7D:4B:14:95,28:3A:4D:46:79:C0,34:6F:24:A8:93:EE,80:32:53:D8:0D:1E
TRANSFER_SERVER = cdimage.ubuntu.com
GSM_CONN_NAME = cht
GSM_APN = internet
GSM_USERNAME = user
GSM_PASSWORD = pass
```

### Environment

- OS: Ubuntu 22.04
- Type: Debian
- Version: 4.0.0~dev270~ubuntu22.04.1

**Comment by xanthein** [comment]
If modify the ```hcitool inq``` in https://github.com/canonical/checkbox/blob/main/providers/base/units/bluetooth/jobs.pxu#L440 to ```bluetoothctl --timeout 10 scan on``` can make the test case pass

**Comment by zongminl** [comment]
Very likely a duplicated issue of #1263, which just have been fixed by PR #1271

**Comment by Hook25** [comment]
Closing as this is a duplicate of the other issue. Should be fixed once we release beta
